app/components/Form/NewContactButton.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button } from "@button-inc/bcgov-theme";
+import { faUserPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useCreateNewContactFormChange } from "mutations/Contact/createNewContactFormChange";
+import { useAddContactToRevision } from "mutations/ProjectContact/addContactToRevision";
+import { useRouter } from "next/router";
+import { getContactFormPageRoute } from "pageRoutes";
+import { createNewContactFormChangeMutation$data } from "__generated__/createNewContactFormChangeMutation.graphql";
+
+interface Props {
+  projectRevisionRowId: number;
+  connectionString: string;
+  projectContactFormId: string;
+  projectId: number;
+  contactIndex: number;
+}
+
+const NewContactButton: React.FC<Props> = ({
+  projectRevisionRowId,
+  connectionString,
+  projectContactFormId,
+  projectId,
+  contactIndex,
+}) => {
+  const router = useRouter();
+  const [addContact, isAddingContact] = useCreateNewContactFormChange();
+  const [addContactMutation] = useAddContactToRevision();
+
+  const clickHandler = (e: React.ChangeEvent<{}>) => {
```

Comment:
Should the project_contact form_change be created now, or when the user submits the contact form?